resources/assets/js/components/modals/edit-songs-form.vue

```diff
@@ -57,6 +57,18 @@
                                 <label>Track</label>
                                 <input type="number" min="0" v-model="formData.track">
                             </div>
+                            <div class="form-row">
```

Comment:
I'd prefer all these info to be non-editable (and not even used), at least for now/